src/app/cpu-utilization-monitor/cpu-observer-list/cpu-observer-list.component.ts

```diff
@@ -32,10 +32,14 @@ export class CpuObserverListComponent {
   }
   /**
    * open dialog and then adding endpoint after the dialog closes
+   * but only if the res is not undefined
+   * response is undefined if the dialog was closed without saving the data
```

Comment:
unterteilung in short / long description beachten.  
ing-form ist hier m.e. fehl am platz.

```suggestion
   * open dialog and add endpoint after the dialog closes
   *
   * only add endpoint if the res is not undefined
   * response is undefined if the dialog was closed without saving the data
```